src/server/internalServer.cpp

```diff
@@ -516,7 +516,7 @@ std::unique_ptr<Response> InternalServer::handle_request(const RequestContext& r
     if (startsWith(request.get_url(), "/raw/"))
       return handle_raw(request);
 
-    if (request.get_url() == "/search")
+    if (startsWith(request.get_url(), "/search"))
```

Comment:
This and related change in `InternalServer::handle_search()` will enable kiwix-serve to serve the intended URL `/search/searchdescription.xml` but it will also make kiwix-serve respond without an error to the following URLs:

- `/searchandarrest?pattern=terrorists`
- `/searchitecture/searchdescription.xml`
- `/search/searchdescription.xml/my/pony/is/afraid/of/unicorns`